### PR TITLE
Some extended basic documentation.

### LINKED ISSUE #567: Go into more details on how you can cluster scope.
A diagram would be nice.  And what to debug if clustering is not working.

### PR DESCRIPTION
**Pull request by tomwilkie**
Fixes #567

### CODE REVIEW

**Comment by tomwilkie** [comment]
@bboreham does this answer the questions you had on Friday?

@peterbourgon would you mind taking a look at this?  Please just push any grammar/spelling/editorial fixes straight to this branch.

**Comment by awh** [comment]
Detail on how probes use `scope.weave.local.` would be helpful

**Comment by bboreham** [comment]
It needs to say, fairly near the beginning, that you should run scope on every host.  That was the key thing that the person I was helping had missed.

File: docs/scope.md [hunk 1]
```diff
@@ -1,10 +1,151 @@
...
+         ,-""'                                                    `.
+        ,'                                                          `.
+        ;            ,'                ','                            `.
+       ,'            ;                 ,'                              `.
+      ,'             ;                 (                    ,           `.
+      ;              `                `.                  ,'           ,'
+      ;               \                 `.                ,'    ,'    ,'
+      '.               '                 `.             ,''.   ,' `--'
+       `"-----"" _.-'' .-'`-.:..___...--' `-._      ,-"'   `-'
+                                      /\      `"""""
```
Comment (tomwilkie) [inline docs/scope.md]: @squaremo I was told you might have some cloud ascii art which looks less like a brain with testicles?

**Comment by peterbourgon** [comment]
LGTM. I fixed the cloud. (Can has VC?)

**Comment by tomwilkie** [comment]
Nice!

**Pull request merged**